src/main/kotlin/com/moshbit/katerbase/MongoHelper.kt

```diff
@@ -29,18 +30,26 @@ abstract class MongoMainEntry : MongoEntry() {
   override fun hashCode(): Int = _id.hashCode()
 
   fun randomId(): String = MongoMainEntry.randomId()
+  fun secureRandomId(): String = MongoMainEntry.secureRandomId()
 
   fun generateId(compoundValue: String, vararg compoundValues: String): String = MongoMainEntry.generateId(compoundValue, *compoundValues)
 
   companion object {
     private val random = Random()
+    private val secureRandom = SecureRandom.getInstanceStrong()
```

Comment:
```suggestion
    private val secureRandom = SecureRandom()
```
We don't want to block here, see https://tersesystems.com/blog/2015/12/17/the-right-way-to-use-securerandom/